hack/validate-imports/allowed-import-names.yaml

```diff
@@ -47,6 +47,8 @@ allowedImportNames:
     - coreosarch
   github.com/googleapis/gnostic/openapiv2:
     - openapi_v2
+  github.com/openshift/api/machine/v1beta1:
+    - machinev1beta1
```

Comment:
I don't think we need this anymore.  I believe it should pass without it as anything under `openshift/api` -> `api` + `apiVersion` which results in `machinev1beta1`.

Can you please remove the old reference in the file too as that is also not in use.  

`github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1`